site2/website-next/docusaurus.config.js

```diff
@@ -254,28 +205,10 @@ module.exports = {
           position: "right",
         },
         {
-          label: "Version",
-          to: "/docs",
-          position: "right",
-          items: [
-            {
-              label: "2.9.0",
-              to: "/docs/2.9.0/",
-            },
-            {
-              label: "2.8.0",
-              to: "/docs/2.8.0/",
-            },
-            {
-              label: "2.7.0",
-              to: "/docs/2.7.0/",
-            },
-            {
-              label: "2.6.0",
-              to: "/docs/2.6.0/",
-            },
-          ],
-        },
+          to: '/download',
```

Comment:
why remove this? it's needed.

Reply:
@urfreespace  On the website weekly meeting yesterday, this change was confirmed